data/software.json

```diff
@@ -1214,6 +1214,24 @@
         "platforms": [],
         "url": null
     },
+    {
+        "categories": [
+            "library"
+        ],
+        "doap": "https://codeberg.org/moxxy/moxxmpp/raw/branch/master/packages/moxxmpp/moxxmpp.doap",
+        "name": "moxxmpp",
+        "platforms": [],
+        "url": "https://codeberg.org/moxxy/moxxmpp/src/branch/master/packages/moxxmpp"
+    },
+    {
+        "categories": [
+            "client"
+        ],
+        "doap": "https://codeberg.org/moxxy/moxxy/raw/branch/master/moxxy.doap",
+        "name": "Moxxy",
+        "platforms": [],
```

Comment:
I added "Android" as a platform.

Reply:
Hi! Sorry for the confusion, but it was alright the way it was before. Platforms should be an empty list if a DOAP file is present, because a DOAP file can list `<os>` and `<programming-language>` tags, which will be processed by our tooling.

Reply:
All three projects' DOAPs provide both OS and programming language information. So, should I just revert the last commit or is there anything else I should do?

Reply:
> So, should I just revert the last commit or is there anything else I should do?

Exactly.

Thanks a lot!